qiskit/providers/models/backendconfiguration.py

```diff
@@ -400,16 +401,41 @@ def acquire(self, qubit: int) -> AcquireChannel:
             raise BackendConfigurationError("Invalid index for {}-qubit systems.".format(qubit))
         return AcquireChannel(qubit)
 
-    def control(self, channel: int) -> ControlChannel:
+    def control(self, qubit: int) -> ControlChannel:
         """
         Return the secondary drive channel for the given qubit -- typically utilized for
         controlling multiqubit interactions. This channel is derived from other channels.
 
         Returns:
             Qubit control channel.
         """
-        # TODO: Determine this from the hamiltonian.
-        return ControlChannel(channel)
+        return ControlChannel(qubit)
+
+    def get_channel_qubits(self) -> List[Any]:
+        """
+        Parse ``channels`` configuration from the backend and return list of Qiskit Pulse channels.
+
+        The ``channels`` contains information of all channels -- their type, purpose and qubits
+        operated on. This method retrieves the associated qubit and returns a list of Qiskit Pulse
```

Comment:
What associated qubit?

Reply:
The method has been updated in [3e6be9b](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/3e6be9b7317b01f9e25fec1a359dbadb33044bde), so the docstrings have been changed as well.